config/locales/en.yml

```diff
@@ -49,6 +49,7 @@ en:
               blank: Income tax cannot be blank
             national_insurance:
               blank: National Insurance cannot be blank
+              not_a_number: National Insurance must be a number, if this does not apply enter 0
             frequency:
```

Comment:
Now we know this, every fully_validatable_integer needs a translation for not_a_number...

Reply:
@starswan we have a separate ticket for this, https://dsdmoj.atlassian.net/browse/EL-566